src/components/SocialBar/SocialBarIcon.tsx

```diff
@@ -20,19 +20,27 @@ export class SocialBarIcon extends React.Component<SocialBarIconProps> {
   static defaultProps = { socialBarTheme: 'light' };
 
   render() {
-    const { children, tooltip, socialBarTheme, ...rest } = this.props;
+    const {
+      children,
+      tooltip,
+      socialBarTheme,
+      className,
+      ...rest
```

Comment:
```suggestion
```

looks unused

Reply:
used in line 46 (was there before this PR)